.circleci/config.yml

```diff
@@ -234,6 +234,30 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  run-minimal-test:
```

Comment:
Any chance we can instead add "run-minimal-test" step to all other tests ? Just after install.

Reply:
Would you suggest:
- installing a minimal version first, then running the minimal tests followed by the installation of the remaining components and finally running additional tests.
or
- doing a complete install, running the minimal test followed by additional tests.

Reply:
The first - after we install the minimal test we can add another environment and run its test ( and maybe repeat the first tests as well - to make sure they don't interfere with each other ).

It would reproduce the case of a user adding a minimal, then installing other components.

I wouldn't mind if the first test leaves a fortio client/server running, and after installing more components and doing more tests checks with a final test that there are no 50x.  We are supposed to do this in the stability cluster, but it would be nice to do it as part of the install test
presubmit as well.